ckanext/qa/i18n/en_AU/LC_MESSAGES/ckanext-qa.po

```diff
@@ -0,0 +1,250 @@
+# English (Australia) translations for ckanext-qa.
+# Copyright (C) 2017 ORGANIZATION
+# This file is distributed under the same license as the ckanext-qa project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ckanext-qa 2.0\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2020-07-09 01:18+0000\n"
+"PO-Revision-Date: 2020-07-01 16:37+0000\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: en_AU\n"
+"Language-Team: en_AU <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+
+#: ckanext/qa/reports.py:83 ckanext/qa/reports.py:89 ckanext/qa/reports.py:150
+msgid "total_stars"
+msgstr "total_ticks"
+
+#: ckanext/qa/reports.py:84 ckanext/qa/reports.py:90 ckanext/qa/reports.py:151
+msgid "average_stars"
+msgstr "average_ticks"
+
+#: ckanext/qa/reports.py:165
+msgid "openness"
+msgstr "data_usability_rating"
+
+#: ckanext/qa/reports.py:166
+msgid "Openness (Five Stars)"
+msgstr "Data usability rating"
+
+#: ckanext/qa/reports.py:167
+msgid ""
+"Datasets graded on Tim Berners Lees' Five Stars of Openness - openly "
```

Comment:
`Lees'` should be `Lee's`. Also, this is an unreasonably long ID.

Reply:
@ThrawnCA This spelling mistake is in the original extension https://github.com/ckan/ckanext-qa/blob/edaf3075ee92c52865f95df3fad046513c834d1b/ckanext/qa/reports.py#L167.
This spelling mistake will not be displayed in the UI as the translation will display the below 'msgstr' below on lines 43-45 which outputs `Tim Berners-Lee's`